docgen/src/guide/Refreshing_cache.md

```diff
@@ -82,7 +82,11 @@ class App extends Component {
   }
 
   componentDidMount() {
-    setInterval(() => this.setState({ refresh: true }), 5000);
+    this.interval = setInterval(() => this.setState({ refresh: true }), 5000);
```

Comment:
WDYT of setting line length in markdown files to something like 60? I prefer having shorter lines when reading something in a post, rather than while writing code

Reply:
Agree, I'm looking for something like that yesterday :)

Reply:
I think we'll need two prettier configs for that, not sure if it's possible already 🤔

Reply:
I think we can, `printWidth` also handle markdown by default.

> https://prettier.io/docs/en/options.html#print-width

Reply:
I meant that we can't have `js: { printWidth: 80 }, md: { printWidth: 55 }` IIRC, but would be good to add

Reply:
Ah ok. Yep would be great to support this kind of configuration!